src/Akeneo/CouplingDetector/Configuration/examples/akeneo_pim-community-dev.php_cd

```diff
@@ -96,172 +98,243 @@ $rules = [
     ),
 ];
 
+// LEGACY can be fixed by moving a class
+// DEPRECATED can be fixed by removing the class when we start to work on last supported version
+$exclusions = [
```

Comment:
imvho, all these Exclusion mean that one somewhere rule is violated
an exclusion requirement means: for one rule, we don't check the following tokens